README.md

```diff
@@ -3,15 +3,121 @@
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
 - **library**, to fetch learning events from various backends, (de)serialize or
-  convert them from various standard formats such as
-  [xAPI](https://adlnet.gov/projects/xapi/),
+    convert them from various standard formats such as
+    [xAPI](https://adlnet.gov/projects/xapi/), or
+    [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 - **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
 - **HTTP API server**, to collect xAPI statements (learning events)
   following the [ADL LRS
   standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree).
 
 ## Quick start guide
 
+### Testing the LRS server with Docker compose
+
+
+> Preliminary notes:
+>
+> 1. [`curl`](https://curl.se), [`jq`](https://stedolan.github.io/jq/) and
+>    [`docker compose`](https://docs.docker.com/compose/) are required to run
+>    some commands of this tutorial. Make sure they are installed first.
+>
+> 2. In order to run the Elasticsearch backend locally on GNU/Linux operating
+>    systems, ensure that your virtual memory limits are not too low and
+>    increase them (temporally) if needed by typing this command from your
+>    terminal (as `root` or using `sudo`): `sysctl -w vm.max_map_count=262144`
+>
+> Reference:
+> https://www.elastic.co/guide/en/elasticsearch/reference/master/vm-max-map-count.html
+
+To bootstrap a test environment on your machine, clone this project first and
+run the `bootstrap` Makefile target:
+
+```
+$ make bootstrap
+```
+
+This command will create required `.env` file (you may want to edit it for your
+test environment), build the Ralph's Docker image and start a single node
+Elasticsearch cluster _via_ Docker compose.
+
+You can check the `elasticsearch` service status using the `status` helper:
+
+```bash
+$ make status
+
+# This is an alias for:
+$ docker compose ps
+```
+
+You may now start the LRS server using:
+
+```
+$ make run-lrs
+```
+
+The server should be up and running at
+[http://localhost:8100](http://localhost:8100). You can check its status using
+the hearbeat probe:
+
+```
+$ curl http://localhost:8100/__heartbeat__
+```
+
+The expected answer should be:
+
+```json
+{"database":"ok"}
+```
+
+If the database status is satisfying, you are now ready to send xAPI statements
+to the LRS:
+
+```
+$ curl -sL https://github.com/openfun/potsie/raw/main/fixtures/elasticsearch/lrs.json.gz | \
```

Comment:
Maybe in a next PR have an archive in ralph repo? WDYT?

Reply:
Or in a dedicated repository to avoid duplicating the same data set again and again?